iroha-lib/model/utils/Utils.cpp

```diff
@@ -0,0 +1,105 @@
+        logger->error(error_message);
+        throw error_message;
+    }
+}
+
+
+void verifyKepair(const iroha::expected::Result<shared_model::crypto::Keypair, std::string>& keypair,
+                  const logger::LoggerPtr& logger,
+                  const fs::path& path,
+                  const std::string& account_name)
+{
+    if (auto error = iroha::expected::resultToOptionalError(keypair))
+    {
+        const auto error_message = fmt::format("Keypair error= {}."
+                                               "\nKeypair path= {}, name= {}.\n",
+                                               error.value(),
+                                               path.string(),
+                                               account_name);
+        logger->error(error_message);
+        throw error_message;
+    }
+}
+
+
+iroha::keypair_t generateKeypair(const std::string& account_name,
+                     const std::string& key_path,
+                     const logger::LoggerManagerTreePtr& log_manager)
+{
+    const auto logger = log_manager->getChild("Main")->getLogger();
+    const auto keys_manager_log = log_manager->getChild("KeysManager")->getLogger();
+    fs::path path(key_path);
+
+    verifyPath(path, logger);
+
+    iroha::KeysManagerImpl manager((path / account_name).string(), keys_manager_log);
+    auto keypair = manager.loadKeys(boost::none);
+
+    verifyKepair(keypair, logger, path, account_name);
+    return iroha::keypair_t(iroha::pubkey_t::from_hexstring(keypair.assumeValue().publicKey()).assumeValue(),
+                            iroha::privkey_t::from_string(toBinaryString(keypair.assumeValue().privateKey())).assumeValue());
+}
+
+
+const std::string getTransactionHash(const iroha::protocol::Transaction& tx)
+{
+    return iroha::hash(tx).to_hexstring();
+}
+
+
+void printTransactionStatus(const std::string& peer_ip,
+                            uint16_t torii_port,
+                            const std::string& tx_hash)
+{
+    const auto status = iroha_lib::GrpcClient(peer_ip, torii_port).getTxStatus(tx_hash);
+    std::cout << "Tx hash=" << tx_hash
+              << "  Status name=" << TxStatus_Name(status.tx_status())
+              << "  Status code=" << status.tx_status()
+              << "  Error code=" << status.error_code()
+              << std::endl;
+}
+
+
+void printTransactionStatuses(const std::string& peer_ip,
+                              uint16_t torii_port,
+                              const std::vector<iroha::protocol::Transaction>& transactions)
+{
+    for (const auto& tx: transactions)
+        printTransactionStatus(peer_ip, torii_port, getTransactionHash(tx));
+}
+
+
+std::string string_to_hex(const std::string& in)
+{
+    std::stringstream ss;
+
+    ss << std::hex << std::setfill('0');
+    for (size_t i = 0; in.length() > i; ++i)
+    {
+        ss << std::setw(2) << static_cast<unsigned int>(static_cast<unsigned char>(in[i]));
```

Comment:
to make its more portable I suggest (after proper include):
```suggestion
        ss << std::setw(2) << static_cast<uint32_t>(static_cast<uint8_t>(in[i]));
```

Reply:
Changed.